site/public/js/server.js

```diff
@@ -474,6 +475,21 @@ function editFilePathRecommendations() {
     registerSelect2Widget('new-file-name', 'material-edit-form');
 }
 
+function showPopup(selector) {
+    $(selector).css('display', 'block');
```

Comment:
jQuery has a `.show()` helper which could make this simpler.  Note: this is just a suggestion, not necessarily a requested change.
```suggestion
    $(selector).show();
```